inc/cookie/namespace.php

```diff
@@ -59,7 +63,13 @@ function on_load() {
 		wp_die( $auth_user );
 	}
 	wp_set_auth_cookie( $auth_user->ID, false );
-	wp_safe_redirect( admin_url() );
+
+	if ( isset( $_GET['redirect_to'] ) ) {
+		$redirect_url = esc_url_raw( $_GET['redirect_to'] );
+	} else {
+		$redirect_url = admin_url();
+	}
+	wp_safe_redirect( $redirect_url );
```

Comment:
I think this needs handling for the multisite case above as well? (i.e. auth on non-main site)

Reply:
Ah good catch

Reply:
@rmccue actually, is this handled by https://github.com/humanmade/delegated-oauth2/pull/17/files#diff-7e8b39e389e795c0c9fea60a95258c640b4cad0e4ed464de554ce87e1d537a4fR53? It already forward all the params when it redirects to the sub site

Reply:
hmm yeah seems like it will work I think; let's merge